app/src/main/java/com/example/whatshouldiread/ui/profile/ProfileFragment.kt

```diff
@@ -0,0 +1,27 @@
+package com.example.whatshouldiread.ui.profile
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.Observer
+import androidx.lifecycle.ViewModelProviders
+import com.example.whatshouldiread.R
+
+class ProfileFragment : Fragment() {
+
+    private lateinit var profileViewModel: ProfileViewModel
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        profileViewModel =
+            ViewModelProviders.of(this).get(ProfileViewModel::class.java)
+        val root = inflater.inflate(R.layout.fragment_profile, container, false)
```

Comment:
`val root = inflater.inflate(R.layout.fragment_profile, container, false)`
The R.layout.fragment_profile references XML that is not there. @petromw

Reply:
Fixed in commit 9dc7836657537c2eb8639a1b7d8326cb16d46580
Added in template XML to fix compile issue